spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -305,6 +305,27 @@ that it could not until explicitly told that it can clean up by providing a meth
 
 If multiple methods are annotated with the same annotation an arbitrary one will be chosen.
 
+[[config-parameters]]
+==== Configuration parameters
+
+This specification has 1 configuration parameter which is retrieved using the MicroProfile Config principals.
+
+     mp.lra.LRA.value=SUPPORTS
+
+The allowed values for the configuration parameter are `SUPPORTS`, the default value, and `NOT_SUPPORTED`.
+When an invalid value is defined, the implementation MUST prohibit the successful startup of the application
+(e.g. through a runtime exception).
+
+When a JAX-RS endpoint, or the containing class, is not annotated with @LRA, but it is called on a MicroProfile LRA compliant runtime,
+the system behaves as if there is an annotation present with the `type()` value taken from the configuration parameter.
```

Comment:
this is `value()` in the LRA annotation - https://github.com/eclipse/microprofile-lra/blob/master/api/src/main/java/org/eclipse/microprofile/lra/annotation/ws/rs/LRA.java#L169

Reply:
So if we go with a boolean valued property then

> the system behaves as if there is an annotation present with the value SUPPORTS if the config property is `true` and NOT_SUPPORTED if it is `false`.

Reply:
yes